test/fail_compilation/fail6451.d

```diff
@@ -1,3 +1,9 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail6451.d(14): Error: `__va_list_tag` is not defined, perhaps `import core.stdc.stdarg;` ?
+---
```

Comment:
This test should use `DISABLED: x86` instead of `version (X86_64) { } else static assert(0)`

Reply:
#9234